src/cli/rustup_mode.rs

```diff
@@ -812,7 +812,7 @@ async fn check_updates(cfg: &Cfg<'_>, opts: CheckOpts) -> Result<utils::ExitCode
                         .unwrap()
                         .tick_chars("⠁⠂⠄⡀⢀⠠⠐⠈ "),
                 );
-                pb.set_message(format!("{} - Checking...", name));
+                pb.set_message(format!("{name} - Checking..."));
```

Comment:
These clippy lints seem to largely originate from your earlier modifications in this PR; squash them back to the respective original commit(s) in this case.